File: 0xxx/04xx/041x/0412.js
/**
 * @param {number} n
 * @return {string[]}
 */
const fizzBuzz = function (n) {
  const result = []

  for (let i = 1; i <= n; i++) {
    let item = ''
    if (!(i % 3)) {
      item += 'Fizz'
    }
    if (!(i % 5)) {
      item += 'Buzz'
    }
    result[i - 1] = item ? item : i.toString()
  }

  return result
}

File: 0xxx/04xx/041x/0412.py
class Solution:
  def fizzBuzz(self, n: int) -> List[str]:
    result = []
    for num in range(1, n + 1):
      result.append(self.transfer(num))
    return result

  def transfer(self, n: int) -> str:
    if n % 15 == 0:
      return 'FizzBuzz'
    elif n % 3 == 0:
      return 'Fizz'
    elif n % 5 == 0:
      return 'Buzz'
    else:
      return str(n)